cypress/e2e/Frontend-FX/FHG-924-FX-filter-cost.cy.js

```diff
@@ -26,20 +24,16 @@ describe('FHG-924-FX-filter-cost.spec - Search by filter - Cost',function(){
         cy.ServiceFilterPage();
         cy.familyHubDetails();
         cy.serviceDetails(); // both filters unchecked
-        cy.get('.govuk-grid-column-two-thirds').contains('Free').should('exist')
-       // 
+
         // apply paid filter    
         cy.costFilter('Paid')
         cy.get('.govuk-grid-column-two-thirds').contains('£').should('exist')
-        cy.get('.govuk-grid-column-two-thirds').contains('Free').should('not.exist')
+        cy.get('.govuk-grid-column-two-thirds').contains('£').should('exist')
```

Comment:
change: we can get rid of this since it's a duplicate of the one above (unless there's a reason to check twice, in which case we could do with a lil comment above as to why)

Reply:
We can remove looks like a c+p error from previous writer